code/game/objects/items/crayons.dm

```diff
@@ -380,6 +380,9 @@
 	if(edible && (M == user))
 		user << "You take a bite of the [src.name]. Delicious!"
 		var/eaten = use_charges(5)
+		if(!reagents.total_volume) //Prevents divsion by zero
+			check_empty()
```

Comment:
I don't think you need to call check_empty(), read the comment below, it's already called in the afterattack().